src/main/java/alexiil/mc/mod/load/Tips.java

```diff
@@ -23,6 +25,19 @@ public class Tips {
     }
 
     public static void load() {
+        File options = new File("options.txt");
+        try (BufferedReader reader = new BufferedReader(new FileReader(options))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(":");
+                if (parts[0].equals("lang")) {
+                    language = parts[1].toLowerCase(Locale.ROOT);
+                }
+            }
+        } catch (IOException ignored) { }
```

Comment:
```suggestion
        } catch (IOException io) {
            CLSLog.warn("Failed to load language from options.txt", io);
        }
```

Just so we don't throw away a potential problem. I would also recommend putting an `if (options.exists())` check before this since it's normal for options.txt to be missing on first launch.